### PR TITLE
Add additional steps to debug repo issues

### PR DESCRIPTION
**Pull request by daxmc99**
Used during #inc-120

### CODE REVIEW

File: content/departments/cloud/technical-docs/operations.md [hunk 1]
```diff
@@ -358,9 +359,11 @@ Once you have identified a repo is constantly failing to be updated/fetched, exe
 ```sh
 mg ssh
 docker exec -it gitserver-0 sh
-cd /data/repos/<repo_name>
-git prune && git fetch
-# look for errors, no output indicates clean repo
+cd /data/repos/<repo_name>/.git
+cat sgm.log
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/operations.md:363]:
```suggestion
cat sgm.log
cat gc.log
```

**Review by michaellzc — APPROVED** [review]

File: content/departments/cloud/technical-docs/operations.md [hunk 2]
```diff
@@ -358,9 +359,12 @@ Once you have identified a repo is constantly failing to be updated/fetched, exe
...
+cat gc.log
+# look for errors and numbers of failures
+# Also run
+git prune && git fetch # check for errors
```
Comment (filiphaftek) [inline content/departments/cloud/technical-docs/operations.md:367]: @daxmc99 the issue I see here is that only cloud team can do it, right? b/c other engineers cannot ssh to instance?

Comment (daxmc99) [reply content/departments/cloud/technical-docs/operations.md:367]: We may provide temporary access in the future to allow other engineers to debug issues (at least this is my understanding)

I agree we shouldn't need ssh access though

Comment (rafax) [reply content/departments/cloud/technical-docs/operations.md:367]: Can we have a GraphQL mutation to inspect the status / remove the offending repo?

**Pull request merged**